sssd_test_framework/utils/realmd.py

```diff
@@ -0,0 +1,100 @@
+"""Manage enrollment in realms."""
+
+from __future__ import annotations
+
+from typing import Any
+
+from pytest_mh import MultihostHost, MultihostUtility
+from pytest_mh.conn import ProcessResult
+
+__all__ = [
+    "RealmdUtils",
+]
+
+
+class RealmdUtils(MultihostUtility[MultihostHost]):
+    """
+    Call commands from realmd
+    """
+
+    def discover(self, domain: str, args: list[Any] | None = None) -> ProcessResult:
+        """
+        Call ``realm discover``  with given arguments.
```

Comment:
```Discover realm.```

Reply:
I'm saying that all these extra words are not necessary. ```Discover Realm.``` is enough and we don't need to add extra verbiage.  ```it's capabilities or with given arguments``` doesn't add any additional clarification on what the method does.

Reply:
rearranged